### DISCUSSION TITLE
Having trouble adding fortawesome plugin

### DISCUSSION THREAD

**Discussion by lordisp** [tailwindlabs/tailwindcss#3708] [Help] [answered]
I've added the plugin according to the TW documentation (also npm install'ed it), Build compiled successful, however the FA css will not included in app.css

tailwind.config.js

``` javascript
plugins: [
        require('@tailwindcss/typography'),
        require('@tailwindcss/forms'),
        require('@tailwindcss/line-clamp'),
        require('@tailwindcss/aspect-ratio'),
        require('tailwindcss-children'),
        require('@fortawesome/fontawesome-free'), <--
    ],
```

app.css
``` css

@import '@fortawesome/fontawesome-free/css/fontawesome';
@import '@fortawesome/fontawesome-free/css/brands';
@import '@fortawesome/fontawesome-free/css/solid';
@import '@fortawesome/fontawesome-free/css/regular';
```

FA path in my repo: `node_modules/@fortawesome/fontawesome-free/css`

Compiling Error:
```
ERROR in ./resources/css/app.css
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/css-loader/dist/cjs.js):
Error: Can't resolve '../webfonts/fa-brands-400.eot' in 'C:\Users\***\localGit\oneup-style\resources\css'
```

Has someone made any experience getting FA back working?

**Comment by royvanv** [comment]
`@fortawesome/fontawesome-free` is not a Tailwind plugin. Your changes to app.css should be enough.

**Reply by lordisp** [reply]
I know that FA is not a tailwind plugin. I'm asking the community here because apparently some other similar challenges have FA with Laravel 8 / Livewire / Tailwind. For some reason the fonts' paths are not recognized. I have found approaches to adapt webjack rules. but acts more as a symptom control rather than a solution. I am currently using the CDN. I want to avoid that for production. So if someone has an idea I would be very grateful for tips.

**Comment by kondratovbr** [comment, marked as answer]
Just dump `fa.css` into your `resources` folder and smack an `@import 'fa.css';` in your `app.css` file. Default PostCSS configuration of Tailwind will even purge those FA classes like it's doing with Tailwind utilities. No need for plugins. Don't forget to dump FA font files into `public/webfonts`.
